app-creator/client/src/widgets/app-root.ts

```diff
@@ -179,19 +239,21 @@ export class AppRoot extends LitElement {
           </div>
 
           <paper-dialog with-backdrop no-cancel-on-outside-click>
+            ${loadingBar}
             <div id="header-content">
               <h2 id="modal-title">Select Template</h2>
               <search-bar
                 placeholder="Search for template (i.e. side panel)"
                 @onsearch=${handleSearch}
               ></search-bar>
             </div>
-            <div id="cards-container">
-              ${filteredTemplates.map(({ markup }) => markup)}
-              ${filteredTemplates.length === 0 ? emptyNotice : nothing}
-            </div>
-            <div class="buttons"></div>
+            ${contentMarkup}
           </paper-dialog>
+
+          <paper-toast
+            id="fetch-error-toast"
+            text="Error fetching templates. Using backup instead."
+          ></paper-toast>
```

Comment:
I'm not sure even showing a message is necessary if the user hasn't noticed anything different. Maybe the error message should be `"Unable to fetch latest templates from the server."`?

Reply:
We can show the snackbar if NODE_ENV='development' and otherwise just log it to the console in production. It might be a useful message for users if the majority of them are developers. Thoughts?